include/z64save.h

```diff
@@ -226,14 +233,14 @@ typedef struct CycleSceneFlags {
 typedef struct SaveOptions {
     /* 0x0 */ u16 optionId;                            // "option_id"
     /* 0x2 */ u8 language;                             // "j_n"
-    /* 0x3 */ s8 audioSetting;                         // "s_sound"
+    /* 0x3 */ u8 audioSetting;                         // "s_sound"
     /* 0x4 */ u8 languageSetting;                      // "language"
     /* 0x5 */ u8 zTargetSetting;                       // "z_attention", 0: Switch; 1: Hold
 } SaveOptions; // size = 0x6
 
 typedef struct SavePlayerData {
     /* 0x00 */ char newf[6];                          // "newf"               Will always be "ZELDA3 for a valid save
-    /* 0x06 */ u16 deaths;                            // "savect"
+    /* 0x06 */ u16 sotCount;                          // "savect"
```

Comment:
`cycle` feels like too general of a word and `cycleCount` makes me think it's referring to some sort of oscillation. I'd rather go with `songOfTimeCount` or `threeDayCycleCount` or `resetThreeDayCycleCount`. But I feel `sot` is a common enough abbreviation of Song of Time? But I could just be very familiar with it. I will say not using `SoT` abbreviation will make the Song of Time Cutscene Docs messy fast:
https://github.com/engineer124/mm/blob/time/src/overlays/actors/ovl_En_Test6/z_en_test6.c

Reply:
The difference here is it's specifically the number of times you've "Saved and returned to Dawn of the First Day", rather than actually played Song of Time. I take the point about test5, although I don't think all lowercase is correct anyway. On JPN it makes sense to be "savect" (as there's no other way to save), on English owl saves somewhat throw a spanner in that.

Reply:
I went with `threeDayResetCount`